packages/dashboard/src/components/queryEditor/iotSiteWiseQueryEditor/modeledDataStreamQueryEditor/modeledDataStreamExplorer/modeledDataStreamTable/modeledDataStreamTable.tsx

```diff
@@ -153,13 +167,11 @@ export function ModeledDataStreamTable({
       }
       preferences={<ModeledDataStreamTablePreferences preferences={preferences} updatePreferences={setPreferences} />}
       ariaLabels={{
-        itemSelectionLabel: (isNotSelected, modeledDataStream) =>
-          isNotSelected
-            ? `Select modeled data stream ${modeledDataStream.name}`
-            : `Deselect modeled data stream ${modeledDataStream.name}`,
+        itemSelectionLabel: ({ selectedItems }, modeledDataStream) =>
+          propertySelectionLabel([...selectedItems] as unknown as ModeledDataStream[], modeledDataStream),
```

Comment:
Is there any way to do this without casting to unknown?

Reply:
yes. updated